x-pack/plugins/cloud_security_posture/public/pages/configurations/findings_flyout/findings_flyout.tsx

```diff
@@ -112,6 +118,24 @@ export const CisKubernetesIcons = ({
   </EuiFlexGroup>
 );
 
+const getFlyoutDescriptionList = (finding: CspFinding): EuiDescriptionListProps['listItems'] =>
+  [
+    finding.resource?.id && {
+      title: i18n.translate(
+        'xpack.csp.vulnerabilities.vulnerabilitiesFindingFlyout.flyoutDescriptionList.resourceId',
```

Comment:
the translations are using vulnerability identifiers, think we can use the identifiers that were being used before, i.e 
`xpack.csp.findings.findingsFlyout.overviewTab.resourceIdTitle`